SPECS/compiler-rt/compiler-rt.spec

```
@@ -0,0 +1,72 @@
+%global maj_ver 12
+%global min_ver 0
+%global patch_ver 1
+
+%global compiler_rt_srcdir %{name}-%{version}.src
+
+Summary:        LLVM compiler support routines
+Name:           compiler-rt
+Version:        %{maj_ver}.%{min_ver}.%{patch_ver}
+Release:        1%{?dist}
+License:        Apache 2.0 WITH exceptions
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Tools
+URL:            https://compiler-rt.llvm.org
+Source0:        https://github.com/llvm/llvm-project/releases/download/llvmorg-%{version}/%{compiler_rt_srcdir}.tar.xz
+BuildRequires:  cmake
+BuildRequires:  llvm-devel = %{version}
+Requires:       llvm
+
+%description
+The compiler-rt project consists of several related runtime libraries for interfacing
+with the clang compiler infrastructure. Builtins is a library for providing low-level
+target-specific hooks required by code generation and other runtime components. Sanitizer
+consists of several sanitizers such as the AddressSanitizer that can include instrumentation
+into compiled binaries. Profile is a library used to collect coverage information and
+BlocksRuntime is an implementation of Apple "blocks" interface.
+
+%prep
+%setup -q -n %{compiler_rt_srcdir}
+
+%build
+# Disable symbol generation
+export CFLAGS="`echo " %{build_cflags} " | sed 's/ -g//'`"
+export CXXFLAGS="`echo " %{build_cxxflags} " | sed 's/ -g//'`"
+
+mkdir -p build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX=%{_prefix}   \
+      -DCMAKE_BUILD_TYPE=Release          \
+      -Wno-dev ..
```

I'd give `%cmake` a shot. This way we have better control over build arguments of our packages and can update them in one place (and it already has `DCMAKE_INSTALL_PREFIX` set the way we want it):
```suggestion
%cmake -DCMAKE_BUILD_TYPE=Release  \
       -Wno-dev ..
```